compose/compose-file/index.md

```diff
@@ -882,6 +882,81 @@ port (a random host port will be chosen).
      - "127.0.0.1:5000-5010:5000-5010"
      - "6060:6060/udp"
 
+### secrets
+
+After you
+[define or reference secrets in the top-level `secrets` configuration](#secrets-configuration-reference),
+grant access to secrets on a per-service basis using the per-service `secrets`
+configuration. Two different syntax variants are supported.
```

Comment:
Maybe for the very very short term. Technical writers tend to avoid the passive voice. Maybe @johndmulhausen can weigh in on that in general. To be honest, I did not read the entire document before contributing this PR, because this PR is very important and late, and this is not normally my content to maintain.